inc/command/class-command.php

```diff
@@ -1054,7 +1081,270 @@ protected function merge_config( array $config, array $overrides ) : array {
 				break;
 			}
 		}
+
 		return $merged;
 	}
 
+	/**
+	 * Bootstraps Docs linter configuration files.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return int
+	 */
+	protected function bootstrap_lintdocs( InputInterface $input, OutputInterface $output ) : int {
+
+		// Copy configuration file for Markdownlint
+		$target_file = getcwd() . '/.markdownlint.jsonc';
+		$template_file = realpath( __DIR__ . '/../../templates/docslint/.markdownlint.jsonc' );
+
+		if ( file_exists( $target_file ) ) {
+			$output->writeln( '<error>Markdownlint configuration file already exists at <root>/.markdownlint-cli2.yml, not overwriting.</error>' );
+		} else {
+
+			$base_command = sprintf( 'cp "%s" "%s" &> /dev/null', $template_file, $target_file );
```

Comment:
Shell arguments need to be escaped. Even though it may not be too obvious here, but `$target_file` is user input as it is based on the current working directory, which can have a folder named `" || rm -rf /` or something along those lines.

Reply:
Copied from previous code. But I'll raise a separate ticket, Thanks.